[WEBSITE-623] Change plugin:jdk-tool by jdk tool (it's not the same) (#2115)
jenkins-infra/jenkins.io: content/doc/administration/requirements/java.adoc

It includes:

* Execution of Java commands from CLI

> * Installation and execution of build steps based on the jdk tool

Reviewer: ```suggestion
* Installation and execution of build steps using JDK managed by JDK tool installers
```
